src/js/components/ClassChoiceButton.js

```diff
@@ -1,16 +1,38 @@
 import React, { Component } from 'react';
 import { saveClass } from '../actions/nextStepCharAction';
+import { connect } from "react-redux"
+import "../../styles/attributes.scss";
 
-require("../../styles/attributes.scss");
-
+@connect((store) => {
+    return {
+        lastStep: store.nextStep
+    };
+})
 export default class ClassChoiceButton extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            cssClass: 'attribute-choice-button'
+        }
+    }
+    
+    getProperActiveElement() {
+        if (this.props.classOfACharacter === this.props.lastStep.class) {
+            return this.state.cssClass = 'attribute-choice-button active-element';
+        }
+        
+        return this.state.cssClass = 'attribute-choice-button';
+    }
+    
     saveClass() {
-        return this.props.dispatch(saveClass(this.props.classOfACharacter));
+        this.props.dispatch(saveClass(this.props.classOfACharacter));
+
+        return this.getProperActiveElement();
     }
 
     render() {
         return (
-            <button class="attribute-choice-button" onClick={this.saveClass.bind(this)}>
+            <button class={this.state.cssClass} onClick={this.saveClass.bind(this)}>
```

Comment:
It's better to bind functions in constructor (performance).

`this.saveClass = this.saveClass.bind(this);`

But if you'll use const instead of state you don't need to bind this function.